### ISSUE TITLE
Win64

### ISSUE THREAD

**Issue by cristgal** [bmatzelle/gow#199]
I downloaded the master-zip and tried to run the tools in WinPE64 (no 32bit compatibility libraries) but I get 'The subsystem needed to support the image type is not present'. Any chance you'll port these for Win64? thanks.

**Comment by entangledloops** [comment]
I just posted about a speed issue with make and suggested building in x64. I'm going to fork and give it a go this weekend. Not sure if this project is being supported anymore? Binaries haven't updated in 2 years, sadly.

**Comment by Alhadis** [comment]
@entangledloops Go for it! :D Speed is never a bad thing (unless you're driving, LEL?)

Hopefully it turns out well!

**Comment by entangledloops** [comment]
Yeah, cygwin / ming are just too heavy for small needs. And who wants to run a slow installer every time you want to add a new src/binary?

**Comment by Alhadis** [comment]
True. But I also use Make heavily in web development (because the world needs another Grunt/Gulp/Gobble/Middleman plugin like it needs another Vietnam war), so it gets recommended reasonably often. =)

**Comment by entangledloops** [comment]
+1 for interesting Make use case.
